api-report/container-loader.api.md

```diff
@@ -152,7 +152,7 @@ export class DeltaManager extends TypedEventEmitter<IDeltaManagerInternalEvents>
     attachOpHandler(minSequenceNumber: number, sequenceNumber: number, term: number, handler: IDeltaHandlerStrategy): void;
     // (undocumented)
     readonly clientDetails: IClientDetails;
-    close(error?: ICriticalContainerError): void;
+    close(reason?: string, error?: ICriticalContainerError): void;
```

Comment:
Switch param order to match `Container`?  It's probably gonna confuse someone if they don't match...

Reply:
I want it to be required argument, so it has to be first. The only reason it's not required is because we put close() on IDeltaManager, where it should not be.